src/include/daos_task.h

```diff
@@ -519,6 +521,34 @@ typedef struct {
 	daos_epoch_range_t	epr;
 } daos_cont_destroy_snap_t;
 
+/** Container snapshot oit oid get args */
+typedef struct {
+	/** Container open handle. */
+	daos_handle_t		 coh;
+	/* Epoch of snapshot for getting oit oid */
+	daos_epoch_t		 epoch;
```

Comment:
These two warnings should have failed the build but did not, and will now show up for all PRs.  Can you please fix it.

Reply:
I see, it have comments, but it should use /** rather than /* , let me fix it, thanks.